tests/testthat/test-logisticregutils.R

```diff
@@ -0,0 +1,34 @@
+context("Logistic Regression: helper functions testing ")
+
+config <- list(
+  `graph.resolution` = dropdownInput('%Question.graph.resolution%' , '1x'),
+  `the.link` = dropdownInput('%Question.Link%' , 'logit'),
+  `model.name` = textInput('%Question.Model Name%', 'my_model'),
+  `used.weights` = checkboxInput('%Question.Use Weights%' , FALSE),
+  `Weight Vec` = dropdownInput('%Question.Weight Vec%'),
+  `X Vars` = listInput('%Question.X Vars%', c('Sepal.Length', 'Petal.Length')),
+  `Y Var` = dropdownInput('%Question.Y Var%', 'Species')
+)
```

Comment:
I would use a simple list without the `***Input` functions, that assign just the defaults. So, this will look something like:

```r
config <- list(
  graph.resolution = '1x',
  the.link = 'logit',
  ...
)
```